src/view/screens/Search/Search.tsx

```diff
@@ -339,6 +347,63 @@ export function SearchScreenInner({
     [setDrawerSwipeDisabled, setMinimalShellMode],
   )
 
+  const sections = React.useMemo(() => {
+    if (!query) return []
```

Comment:
This seems suspicious — previously we never had an empty array of pager items. I'm not sure the pager supports zero items.

Reply:
Hm I see, this shouldn't happen anyway because then the conditionals below would render a different codepath.